src/presenters/title-presenter.php

```diff
@@ -37,17 +37,21 @@ public function __construct( String_Helper $string ) {
 	 * Returns the document title.
 	 *
 	 * @param Indexable_Presentation $presentation The presentation of an indexable.
+	 * @param bool                   $output_html  Optional. Whether or not to output HTML. Defaults to true.
 	 *
 	 * @return string The document title tag.
 	 */
-	public function present( Indexable_Presentation $presentation ) {
+	public function present( Indexable_Presentation $presentation, $output_html = true ) {
 		$title = $this->replace_vars( $presentation->title, $presentation );
 		$title = $this->filter( $title, $presentation );
 		$title = $this->string->strip_all_tags( \stripslashes( $title ) );
 		$title = \trim( $title );
 
 		if ( \is_string( $title ) && $title !== '' ) {
-			return '<title>' . \esc_html( $title ) . '</title>';
+			if ( $output_html ) {
+				return '<title>' . \esc_html( $title ) . '</title>';
+			}
+			return \esc_html( $title );
```

Comment:
I don't think you should `esc_html` here.

Reply:
In https://developer.wordpress.org/reference/functions/wp_title/ it's echo'd immediately after the filter with no escaping so I don't think WP is escaping anything itself.

Reply:
Aargh that's so ugly. OK.